### PR TITLE
show more preview text in wider forum history page

### PR DESCRIPTION
**Pull request by Jamiras**
Increases truncation from 90 characters to 120.

### CODE REVIEW

File: public/forumposthistory.php [hunk 1]
```diff
@@ -95,7 +98,7 @@
         echo "<a href='/viewtopic.php?t=$forumTopicID&c=$commentID#$commentID'>$forumTopicTitle</a>";
         echo " <span class='smalldate $tooltipClassName' $titleAttribute>$postedAt</span>";
         echo "<div class='comment text-overflow-wrap'>";
```
[the remark this answers is not in the document]
Comment (Jamiras) [reply public/forumposthistory.php:100]: The `line-clamp-2` doesn't seem to do anything. Even at XL, a small portion of text wraps at 130:
image

Even at 120, things with lots of capital letters wrap:
image

But 120 doesn't wrap for _most_ things, which is why I chose it.

[the remark this answers is not in the document]
Comment (Jamiras) [reply public/forumposthistory.php:100]: `line-clamp-2` does affect smaller views, and seems to provide the ellipsis automatically. 
image

What do you think about using `line-clamp-2` and making the message limit 99999?
image

`line-clamp` and `line-clamp-1` don't seem to be valid tags.

[the remark this answers is not in the document]
Comment (wescopeland) [reply public/forumposthistory.php:100]: Yes, `line-clamp` forces the text to automatically truncate at a certain line number and tacks on a magic ellipsis with CSS.

`line-clamp-none` is probably valid. We could even choose how much to clamp on different breakpoints like `line-clamp-2 md:line-clamp-3 lg:line-clamp-4` (not suggesting this literally, just an example)

`line-clamp-2` and an increased message limit makes sense!

**Review by wescopeland — APPROVED** [review]

File: public/forumposthistory.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jamiras) [reply public/forumposthistory.php:100]: Going to leave this at 120 for now. We can look at doing something smarter in the future.

**Pull request merged**